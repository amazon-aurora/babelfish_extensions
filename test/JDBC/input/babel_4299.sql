select ''+'ps';
GO

select 'rs' + '';
GO

select '' + '';
GO

select '' + NULL;
GO

select NULL + '';
GO

set quoted_identifier off
GO
select ""+'s';
GO

set quoted_identifier off
GO
select 'rs' + "";
GO

set quoted_identifier off
GO
select '' + "";
GO

set quoted_identifier off
GO
select "" + NULL;
GO

set quoted_identifier off
GO
select NULL + "";
GO

set quoted_identifier off
GO
select ""+"s";
GO

set quoted_identifier off
GO
select "pr" + "";
GO

set quoted_identifier off
GO
select "" + "";
GO

set quoted_identifier off
GO
select "" + "";
GO
