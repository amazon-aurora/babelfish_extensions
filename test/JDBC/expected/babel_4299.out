select ''+'ps';
GO
~~START~~
varchar
ps
~~END~~


select 'rs' + '';
GO
~~START~~
varchar
rs
~~END~~


select '' + '';
GO
~~START~~
varchar

~~END~~


select '' + NULL;
GO
~~START~~
varchar
<NULL>
~~END~~


select NULL + '';
GO
~~START~~
varchar
<NULL>
~~END~~


set quoted_identifier off
GO
select ""+'s';
GO
~~START~~
varchar
s
~~END~~


set quoted_identifier off
GO
select 'rs' + "";
GO
~~START~~
varchar
rs
~~END~~


set quoted_identifier off
GO
select '' + "";
GO
~~START~~
varchar

~~END~~


set quoted_identifier off
GO
select "" + NULL;
GO
~~START~~
varchar
<NULL>
~~END~~


set quoted_identifier off
GO
select NULL + "";
GO
~~START~~
varchar
<NULL>
~~END~~


set quoted_identifier off
GO
select ""+"s";
GO
~~START~~
varchar
s
~~END~~


set quoted_identifier off
GO
select "pr" + "";
GO
~~START~~
varchar
pr
~~END~~


set quoted_identifier off
GO
select "" + "";
GO
~~START~~
varchar

~~END~~


set quoted_identifier off
GO
select "" + "";
GO
~~START~~
varchar

~~END~~

